README.md

```diff
@@ -242,6 +242,63 @@ Autoscaling rules are applied automatically during the next deploy, though may b
 ```shell
 dokku scheduler-kubernetes:autoscale-rule-apply $APP PROC_TYPE
 ```
+### Persistent Volume Claims (pvc) 
+
+Pods use pvcs as volumes.  For volumes that support multiple access modes, the user specifies which mode is desired when using their claim as a volume in a Pod. [See supported access modes by providers](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes)
+
+```shell
+# create a pvc
+dokku scheduler-kubernetes:add-pvc --name $NAME --access-mode $MODE --storage "100" --ns $NAMESPACE [--storage-class-name $CLASS]
```

Comment:
Probably don't need to have `--name` as a flag, can be the first arg. As well, `--ns` should be long-form (`--namespace`) and should have a default of default?

Reply:
What is the default storage class name? Can we default access mode as well?

Reply:
default is dependent on the provider but we set it to ""

Reply:
Lets do that.